src/resolver/fileLookup.ts

```diff
@@ -100,43 +102,56 @@ export function fileLookup(props: ILookupProps): ILookupResult {
   const exists = fileExists(resolved);
   if (exists) {
     const stat = fs.lstatSync(resolved);
-    if (stat.isDirectory) {
-      isDirectory = true;
-
-      let monorepoModulesPaths;
-      let tsConfigAtPath: TsConfigAtPath;
-
-      // only in case of a directory
-      const packageJSONPath = path.join(resolved, 'package.json');
-      if (fileExists(packageJSONPath)) {
-        const useLocalMain = !/node_modules/.test(packageJSONPath);
-        const packageJSON = require(packageJSONPath);
-        const entry = getFolderEntryPointFromPackageJSON({ json: packageJSON, useLocalField: useLocalMain });
-
-        if (useLocalMain && packageJSON['local:main']) {
-          const _monoModules = path.resolve(resolved, 'node_modules');
-          if (fileExists(_monoModules)) {
-            monorepoModulesPaths = _monoModules;
+    isDirectory = stat.isDirectory();
+
+    if (isDirectory) {
+      if (props.inBaseUrl !== true) {
+        let monorepoModulesPaths;
+        let tsConfigAtPath: TsConfigAtPath;
+        // only in case of a directory
+        const projectRoot = path.join(resolved, "../../../"); // TODO: get the actual projectRoot
```

Comment:
This doesn't look right ;-)

Reply:
Yeah.  I'm beginning to  think figuring out whether or not something is a module shouldn't be done inside fileLookup.  Maybe paths lookup would be better...  Not sure.  But all the other lookups eventually reference fileLookup, so changes here seem to have too much effect.